src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String colour;
```

Comment:
It's a good practice to provide getters for the fields of a class. This allows for controlled access to the class's fields.